docs/partials/demo.md

```diff
@@ -27,6 +27,16 @@ The dropdown can be closed with the following actions:
     - `spacebar`
     - `esc`
 
+## Accessibility requirements
+
+All instances of `auro-dropdown` must have a valid `aria-label` or `aria-labelledby` attribute. There are three options available to meet this requirement. Only one of these options should be implemented.
```

Comment:
This is a duplicate of the content in `demo.md` - we should only have one.

Reply:
This is the template that our compiled ./demo/demo.md builds from. This is where the change is made manually.